packages/frontend/src/locales/en.json

```
@@ -11,6 +11,26 @@
       "panelHeading": "Administrators",
       "usernameLabel": "Username"
     },
+    "connectorCreate": {
+      "connectorModalTitle": "Create new connector",
+      "doneLabel": "Done",
+      "doneSuccessDescription": "Connector created!",
+      "doneFailedDescription": "Connector creation failed!",
+      "notifications": {
+        "connectorCreated": "Connector created successfully!"
+      },
+      "parametersLabel": "Parameters",
+      "parametersConnectorName": "Connector Name:",
+      "parametersConnectorNamePlaceholder": "ComputerScience Channel",
+      "selectTypeLabel": "Type",
+      "selectTypeContent": "You want to create a new connector? Great! First, choose your wanted type from the list below:",
```

```suggestion
      "selectTypeContent": "You want to create a new connector? Great! First, choose your desired type from the list below:",
```
translation suggestion